### DISCUSSION TITLE
Google工程師曝AI聊天機器人有知覺　「像7、8歲孩子」

### DISCUSSION THREAD

**Discussion by YuHengChen54** [1101-GS3073-C/Python-Community-GS3073-C#442] [General]
電影情節真實上演？　Google工程師曝AI聊天機器人有知覺　「像7、8歲孩子」
這種AI自己產生的意識真的很酷，但這背後的倫理問題，現在還是很難找到一個平衡。
參考網址:https://www.upmedia.mg/news_info.php?Type=3&SerialNo=146873

**Comment by lx3211** [comment]
有人說apple的siri也有不同的個性
不知道是不是隨機的固定設定
我感覺我的Siri都很冷漠
講話都很冷酷

**Comment by Chenszuhong** [comment]
確實很難找到平衡，畢竟他們有思想之後就像似人類

**Comment by limitx0** [comment]
像不像人每個人的感覺都不同
但是未來 AI 一定可以發展得越來越像人

**Comment by Ben6uo** [comment]
感覺以後AI發展到能自訂對話心理年齡
